front/components/data_source_view/DataSourceViewSelector.tsx

```diff
@@ -173,7 +162,7 @@ export function DataSourceViewsSelector({
   } else {
     return (
       <Tree isLoading={false}>
-        {groupManaged && (
+        {managedDsv.length > 0 && useCase === "assistantBuilder" && (
```

Comment:
So we always group now?

Reply:
Why do we remove the sorting of the data source view?

Reply:
It was a really weird UX to have some vaults grouped and some others not

Reply:
The ordering was mainly on providers type, doesn't make much sense if we group

Reply:
Then why not using the new `managedDsv` on line 173?

Reply:
Seen IRL.